src/test/e2e-emulated/contracts/base-trait-constant-override-2.tact

```diff
@@ -0,0 +1,21 @@
+trait OverrideBaseTraitConstant with BaseTrait {
+    override const storageReserve: Int = 100;
+}
+
+trait OverrideBaseTraitConstantWithVirtual {
+    override virtual const storageReserve: Int = 100;
```

Comment:
this should not compile: `override` is an error here, since you don't override anything

Reply:
Yeah, we're not actually checking this right now (https://github.com/tact-lang/tact/issues/1713)
I'll add a fix to that PR too

Reply:
Done